autogen/logger/logger_factory.py

```diff
@@ -0,0 +1,14 @@
+from typing import Dict
+from autogen.logger.base_logger import BaseLogger
+from autogen.logger.sqlite_logger import SqliteLogger
+
+__all__ = ("LoggerFactory",)
+
+
+class LoggerFactory:
+    @staticmethod
+    def get_logger(logger_type: str = "sqlite", config: Dict = {}) -> BaseLogger:
```

Comment:
Need to handle case when `config` is None as it can be passed in from `start_logging`.